services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/BasePreservedKeyChecker.java

```diff
@@ -16,6 +21,16 @@ public boolean isKeyPreserved(String key) {
     return IS_OH_PREFIXED.test(key);
   }
 
+  /**
+   * Details of determining if a key is preserved for the given {@link TableDto} is hidden in the
+   * implementation of aspect that associates with the annotation {@link TblPropsEnabler}
+   */
+  @Override
+  @TblPropsEnabler
```

Comment:
could this annotation be more specific to better indicate that is may override the return value to true if table prop is enabled by feature?

e.g.

@OverrideIfTablePropsEnabled

Reply:
couldn't really think of a better name ... I will avoid "Override" since there's `@Override` annotation there.